src/GusApi/Type/Request/GetBulkReport.php

```diff
@@ -1,44 +1,27 @@
 <?php
 
+declare(strict_types=1);
+
 namespace GusApi\Type\Request;
 
-class GetBulkReport
-{
-    /**
-     * @var string
-     */
-    protected $pDataRaportu;
+use GusApi\ParamName;
 
-    /**
-     * @var string
-     */
-    protected $pNazwaRaportu;
+final class GetBulkReport implements RequestInterface
+{
+    private string $pDataRaportu;
```

Comment:
Since we are on PHP 8.x, we can use https://wiki.php.net/rfc/constructor_promotion.

I recommend using https://github.com/rectorphp/rector to migrate the code to newer versions.